pkg/controller/traceflow/controller.go

```diff
@@ -62,12 +62,13 @@ const (
 
 	// String set to TraceflowStatus.Reason.
 	traceflowTimeout = "Traceflow timeout"
+
+	// Traceflow timeout period.
+	defaultTimeoutDuration = 20 * time.Second
```

Comment:
maybe we could move the default timeout to pkg/apis/ops/v1alpha1/ so antctl, controller, and agent can use same default value?

Reply:
Sounds a good idea, but do you want to add to types.go?